ios/fastlane/.env.default

```diff
@@ -0,0 +1,9 @@
+// .env.default
+XCODE_SELECT_PATH = /Applications/Xcode_15.0.1.app
```

Comment:
May be use `Xcode_15.1.app` as its the latest available in GitHub actions runners?